CMakeLists.txt

```diff
@@ -112,9 +112,11 @@ macro(create_solo12_demo source)
   target_link_libraries(${demo_name} ${PROJECT_NAME})
   target_compile_definitions(
     ${demo_name}
-    PUBLIC CONFIG_SOLO12_YAML="${PROJECT_SOURCE_DIR}/demos/config_solo12.yaml")
+    PUBLIC CONFIG_SOLO12_YAML="${PROJECT_SOURCE_DIR}/demos/config_solo12.yaml"
+    PUBLIC CONFIG_TESTBENCH_YAML="${PROJECT_SOURCE_DIR}/demos/config_testbench.yaml")
   install(TARGETS ${demo_name} DESTINATION bin)
 
 endmacro(create_solo12_demo source)
 
 create_solo12_demo(demo_solo12)
+create_solo12_demo(demo_testbench)
```

Comment:
Maybe we should rename `create_solo12_demo` to `create_demo` here when it's for non-solo12 robots.